src/pages/PlanPage/subRoutes/InvoiceDetailsPage/sections/InvoiceOverview.tsx

```diff
@@ -29,17 +41,22 @@ const InvoiceOverview = ({ isPaid, invoice, dueDate }) => {
           <tr>
             <td className="pr-2">Date due</td>
             <td>
-              {format(
-                utcToZonedTime(fromUnixTime(dueDate), 'UTC'),
-                'MMMM do, yyyy',
-                { timeZone: 'UTC' }
-              )}
+              {dueDate &&
```

Comment:
We can wrap the whole td in the `dueDate` check. Also if we're on board with what Nick mentioned on my other PR, let's change these two sections to ternary statements.